opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -310,7 +311,7 @@ def start_as_current_span(
         is equivalent to::
 
             span = tracer.start_span(name)
-            with tracer.use_span(span, end_on_exit=True):
+            with tracer.use_span(span, end_on_exit=True, record_exception=True):
```

Comment:
Result of going back and forth between having record_exception as True vs False by default.

Reply:
There's no mention in the example that actually demonstrates what `record_exception` does. It seems like the example is to just show that the `start_as_current_span` and the `start` + `use` is equivalent. This might confuse people that they MUST set `record_exception` for the two behaviours to be the same. So unless we add another example outlining the purpose of `record_exception`, I wouldn't include this explicitly.